view/adminhtml/web/js/mass-action.js

```diff
@@ -275,6 +281,15 @@ define(
                     }
                 },
 
+                /**
+                 * @protected
+                 */
+                _getPPSExportMode: function () {
```

Comment:
ik zou het een andere naam geven voor duidelijkheid, aangezien die een bool teruggeeft, bv _isPPSExportMode of misschien beter in deze context _usePPSExportMode of _isSetToPPSExportMode